src/sentry/incidents/subscription_processor.py

```diff
@@ -129,6 +129,8 @@ def process_update(self, subscription_update):
 
         aggregation = QueryAggregations(self.alert_rule.aggregation)
         aggregation_name = query_aggregation_to_snuba[aggregation][2]
+        if len(subscription_update["values"]["data"]) > 1:
```

Comment:
I'm guessing you're just relying on the `IndexError` here to validate that the result includes at least one row?

Reply:
The schema will validate that part, we don't really need to check it here.